src/core/caniuse.js

```diff
@@ -105,22 +105,16 @@ function getNormalizedConf(conf) {
   }
   const caniuseConf = { ...DEFAULTS, ...conf.caniuse };
   if (Array.isArray(caniuseConf.browsers)) {
-    caniuseConf.browsers = caniuseConf.browsers
-      .map(b => b.toLowerCase())
-      .filter(isValidBrowser);
+    for (const browser of caniuseConf.browsers)
+      if (!BROWSERS.has(browser)) {
+        pub(
```

Comment:
Collect the offending browsers and publish a single warning? 🙏

Reply:
![image](https://user-images.githubusercontent.com/8426945/54752798-59674b80-4c05-11e9-9b69-bfbcf90dde42.png)
Fixed